kachaka_descriptionパッケージのrobot_display.launch.xmlが起動できない。

## 現象

`kachaka_description`パッケージのrobot_display.launch.xmlが起動できない。


```bash
ros2 launch kachaka_description robot_display.launch.xml
[INFO] [launch]: All log files can be found below /home/ubuntu/.ros/log/2025-03-16-09-50-02-111690-4fba6cdbc4ed-3223
[INFO] [launch]: Default logging verbosity is set to INFO
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Caught multiple exceptions when trying to load file of format [xml]:
 - FileNotFoundError: [Errno 2] No such file or directory: '/home/ubuntu/dev_ws/install/kachaka_description/share/kachaka_description/launch/robot_description.launch.xml'
 - FileNotFoundError: [Errno 2] No such file or directory: '/home/ubuntu/dev_ws/install/kachaka_description/share/kachaka_description/launch/robot_description.launch.xml'
```

## 環境

- Ubuntu 22.04
- kachaka-api v3.9.5

## 調査

<https://github.com/pf-robotics/kachaka-api/blob/v3.9.5/ros2/kachaka_description/launch/robot_display.launch.xml#L7-L10>にて存在しないファイル（`robot_description.launch.xml`）を呼び出しています。`robot_description.launch.py`を呼び出すのが正しいように思います。

--- a/ros2/kachaka_description/launch/robot_display.launch.xml
+++ b/ros2/kachaka_description/launch/robot_display.launch.xml
@@ -4,7 +4,7 @@
   <arg name="frame_prefix" default="/" />
   <arg name="use_gui" default="true" />
 
-  <include file="$(find-pkg-share kachaka_description)/launch/robot_description.launch.xml" >
+  <include file="$(find-pkg-share kachaka_description)/launch/robot_description.launch.py" >
       <arg name="namespace" value="$(var namespace)" />
       <arg name="frame_prefix" value="$(var frame_prefix)" />
   </include>

--- a/ros2/kachaka_description/package.xml
+++ b/ros2/kachaka_description/package.xml
@@ -15,7 +15,9 @@
   <exec_depend>launch_ros</exec_depend>
   <exec_depend>launch_xml</exec_depend>
   <exec_depend>joint_state_publisher</exec_depend>
+  <exec_depend>joint_state_publisher_gui</exec_depend>
   <exec_depend>robot_state_publisher</exec_depend>
+  <exec_depend>rviz2</exec_depend>
   <exec_depend>urdf</exec_depend>
   <exec_depend>xacro</exec_depend>
   <test_depend>ament_lint_auto</test_depend>

